sdk/php/runtime/template/README.md

```diff
@@ -311,3 +311,48 @@ class Example
      // ...
 }
 ```
+
+#### Directories and Files
+
+Directories and Files have some additional metadata that can be specified
+
+##### Defaults
```

Comment:
You could also link to the official doc page for more info

https://docs.dagger.io/manuals/developer/functions/#directories-and-files